### ISSUE TITLE
The program run.py is missing essential files and python classes.

### ISSUE THREAD

**Issue by zzysjtuiwct** [sunnweiwei/MixCL#2]
1. No definition of 'Data' import from 'driver.train_bart' in line 680 of run.py can be found in this repository.
2. JSON files: 'wizard_wiki_full.json', 'train_knowledge_span_new.json', 'train_knowledge_span.json', and 'train_knowledge_ner.json' can not be found in the provided dataset.
3. could you provide requirements.txt for this project?

**Comment by TokisakaReijiKun** [comment]
> 1. No definition of 'Data' import from 'driver.train_bart' in line 680 of run.py can be found in this repository.
> 2. JSON files: 'wizard_wiki_full.json', 'train_knowledge_span_new.json', 'train_knowledge_span.json', and 'train_knowledge_ner.json' can not be found in the provided dataset.
> 3. could you provide requirements.txt for this project?

do you have solved it?

**Comment by stysky** [comment]
Please provide the relevant documents, I need to reproduce the experiment

**Comment by TokisakaReijiKun** [comment]
> Please provide the relevant documents, I need to reproduce the experiment

May I ask if you have successfully reproduced it

**Comment by stysky** [comment]
> > Please provide the relevant documents, I need to reproduce the experiment
> 
> May I ask if you have successfully reproduced it

No, the experimental steps in this paper are brief, the open source code is incomplete, and manual completion may be required.
